source/reference/minio-mc/minio-client-settings.rst

```diff
@@ -25,6 +25,8 @@ Host Credentials
 Use this setting to add a temporary alias to use for `mc` commands.
 For example, for use with scripting.
 
+The temporary alias uses the ``s3v4`` API.
```

Comment:
I don't know what `s3v4` means, is there something useful to link to? Is this our API or one from AWS?

Reply:
It's more of a signature than API